docs/source/deploypeer/peerchecklist.md

```diff
@@ -0,0 +1,406 @@
+
+If you plan to use Prometheus [metrics](#metrics) to monitor your peer, you must configure the operations service here.
+
+In the unlikely case where two peers are running on the same node, you need to modify the `listenAddress`: for the second peer to use a different port. Otherwise, when you start the second peer, it will fail to start, reporting that the bind address is already in use.
+
+- **`operations.listenAddress:`** (Required when using the operations service.) Specify the address and port of the operations server.
+- **`operations.tls.cert.file*:`** (Required when using the operations service). Can be the same file as the `peer.tls.cert.file`.
+- **`operations.tls.key.file*:`** (Required when using the operations service). Can be the same file as the `peer.tls.key.file`.
+- **`operations.tls.clientAuthRequired*:`** (Required when using the operations service). Must be set tp `true` to enable mutual TLS between the client and the server.
+- **`operations.tls.clientRootCAs.files*:`** (Required when using the operations service). Similar to the [peer.tls.clientRootCAs.files](#tls), it contains a list of client root CA certificates that can be used to verify client certificates. If the client enrolled with the peer organization CA, then this value is the peer organization root CA cert.
+
+## metrics
+
+```
+metrics:
+    # metrics provider is one of statsd, prometheus, or disabled
+    provider: disabled
+    # statsd configuration
+    statsd:
+        # network type: tcp or udp
+        network: udp
+
+        # statsd server address
+        address: 127.0.0.1:8125
+```
+By default this is disabled, but if you want to monitor the metrics for the peer, you need to choose either `statsd` or `Prometheus` as your metric provider. `Statsd` uses a "push" model, pushing metrics from the peer to a `statsd` endpoint. Because of this, it does not require configuration of the operations service itself. See the list of [Available metrics for the peer](../metrics_reference.html#peer-metrics).
+
+- **`provider`:** (Required to use `statsd` or `Prometheus` metrics for the peer.) Because Prometheus utilizes a "pull" model there is not any configuration required from the Fabric CA server side. Rather, Prometheus will sends requests to the operations URL to poll for available metrics.
+- **`address:`** (Required when using `statsd`.) When `statsd` is enabled, you will need to configure the hostname and port of the server.
```

Comment:
```suggestion
- **`address:`** (Required when using `statsd`.) When `statsd` is enabled, you will need to configure the hostname and port of the statsd server so that the peer can push metric updates.
```